core/src/main/java/org/apache/accumulo/core/spi/crypto/AESCryptoService.java

```diff
@@ -200,6 +207,15 @@ public void setEncFek(byte[] encFek) {
       this.encFek = encFek;
     }
 
+    @Override
+    public void close() throws Exception {
+      cryptoServiceName = null;
+      cryptoServiceVersion = null;
+      keyManagerVersion = null;
+      kekId = null;
+      Arrays.fill(encFek, (byte) 0);
+    }
+
```

Comment:
These changes look good so far. I like using AutoCloseable, that is a good idea.